src/main/resources/l10n/JabRef_fa.properties

```diff
@@ -1149,7 +1149,7 @@ Settings=
 
 Shortcut=
 
-Show/edit_BibTeX_source=
+Show/edit_%0_source=Montrer/�diter_le_source_BibTeX
```

Comment:
1. this file is not in `utf-8` thus the `é` makes problems with the python script (it should make problems with JabRef too as they're loaded with `utf-8` in Java)
- isn't this french?